### ISSUE TITLE
Peer Review - Mac

### ISSUE THREAD

**Issue by jj-zhang** [OpenLiberty/draft-guide-reactive-service-testing#18]
Peer Review: review to be done by a peer member. 

Functionality
---
- [x] Check that the guide has working instructions and sample code
- [ ] Test the guide on 3 platforms: Mac, Win, Linux
- [x] If any URL visits, try them on different browsers: Firefox, Chrome, Safari
- [x] If any URL visits, try `curl` command where applicable 

Formatting & Presentation
---
- [x] Check the quality of code according to the best coding practices
    - [x] Ensure that command substitutions use the `$(cmd)` format and not ``` `cmd` ```
    - [x] Ensure that the console output of the integrated tests do not include the `[INFO]` tab
- [x] Check the quality and presentation of guide according to [Structure and Style Guideline](https://github.com/OpenLiberty/draft-guides-template/wiki/Guidelines-for-Structure-and-Styles)
- [x] Check the consistency of guide with template and other guides
- [x] Examples with the right outcomes are provided
- [x] Example with the wrong outcomes, if any, are provided
- [x] Check that all licensing statements are properly stated in all files
- [x] Check that the `pom.xml`, `server.xml`, etc files are clean
- [x] Check that the directories are properly structured
- [x] Ensure that the guide is using the latest version of the `liberty-maven-plugin` or `liberty-gradle-plugin` where applicable
- [x] Check that some of these `page-tags` are used in a guide: `MicroProfile, Maven, Docker, Kubernetes, Gradle, Java EE, Security, Cloud`. Only these tags are visible on the website. Latest list [here](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/_includes/visible-tags.liquid). 
- [x] Check the `attribution` statement is accurate for the guide
- [x] Verify the `travisTest.sh` script, if any, is accurate and consistent with other guides

Peer Testing: tests to be done by a peer member.

Guide’s contributor’s (if available, otherwise peer tester’s) responsibility: 
- [x] Additional tests where applicable: 
    - [x] Define test coverage and review with team (including guide contributor, if available)
    - [x] Define detail test cases 
    - [x] Consider corner cases targeting the specific guide
    - [ ] Consider different platform tests, ie, Mac, Windows, Linux
    - [x] Consider corner cases UI tests
    - [x] Consider testing URL on all browsers, ie, FF, Chrome, Safari
    - [x] Consider testing the `curl` command for URL visits
- [x] Consider building with both Maven and Gradle build tools 
- [x] Testing with different IDEs, ie, Atom, Eclipse (Optional: VS.code, IntelliJ, Microclimate)
- [x] Run Acrolinx Checker on draft (above 70 score approximately)
- [x] Consider SEO title and description for the guides 
- [x] Ensure automated test is enable, set up with Travis CI, and able to schedule tests 
- [x] Run `diff -r start/ finish/` and there's no differences
- [x] Ensure that the automation tests are able to run when PR is created

Peer Tester’s responsibility: 
- [x] Check the appearance of the guide on test site for the following items: 
    - [x] Table of contents 
    - [x] Headings
    - [x] Paragraphs
    - [x] code snippets
    - [x] outputs
    - [x] links
    - [x] hotspots 
- [x] Test the guide end-to-end with working instruction and sample code
- [x] Perform all the defined test cases

**Comment by jj-zhang** [comment]
### Testing using the Kafka producer client
- [ ] Got the warning when running `mvn verify` in `start/system` directory.
```
[WARNING] /Users/Darren.Zhang@ibm.com/ibm-guide/draft-guide-reactive-service-testing/start/system/src/main/java/io/openliberty/guides/system/SystemService.java:[51,35] package osMean does not exist
```

- [ ] Got an error as well.
```
[INFO] Press the Enter key to run tests on demand. To stop the server and quit dev mode, use Ctrl-C or type 'q' and press the Enter key.
/Users/Darren.Zhang@ibm.com/ibm-guide/draft-guide-reactive-service-testing/start/system/src/main/java/io/openliberty/guides/system/SystemService.java:51: error: package osMean does not exist
                        new osMean.getSystemLoadAverage())));
                                  ^
1 error
```

### Testing using the Kafka consumer client

- [ ] Got an error when running `mvn liberty:dev`
```
INFO] Listening for transport dt_socket at address: 7777
[INFO] Launching defaultServer (Open Liberty 20.0.0.4/wlp-1.0.39.cl200420200401-1714) on OpenJDK 64-Bit Server VM, version 11.0.3+7 (en_CA)
[INFO] CWWKM2013I: The file /Users/Darren.Zhang@ibm.com/ibm-guide/draft-guide-reactive-service-testing/start/inventory/target/liberty/wlp/usr/servers/defaultServer/logs/messages.log being validated does not exist.
[INFO] [AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
[INFO] [AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /Users/Darren.Zhang@ibm.com/ibm-guide/draft-guide-reactive-service-testing/start/inventory/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/liberty-plugin-variable-config.xml
[INFO] [AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://192.168.100.167:9085/health/
[INFO] **[err]** SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[INFO] **[err]** SLF4J: Defaulting to no-operation (NOP) logger implementation
[INFO] **[err]** SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://192.168.100.167:9085/
[INFO] [AUDIT   ] CWWKZ0001I: Application inventory started in 1.742 seconds.
[INFO] [AUDIT   ] CWWKF0012I: The server installed the following features: [cdi-2.0, concurrent-1.0, jaxrs-2.1, jaxrsClient-2.1, jndi-1.0, json-1.0, jsonb-1.0, jsonp-1.1, mpConfig-1.3, mpHealth-2.1, mpReactiveMessaging-1.0, mpReactiveStreams-1.0, servlet-4.0].
[INFO] [AUDIT   ] CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 4.118 seconds.
[INFO] CWWKM2015I: Match number: 1 is [2020-04-13, 14:06:07:116 EDT] 00000023 com.ibm.ws.kernel.feature.internal.FeatureManager            A CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 4.118 seconds..
[INFO] Press the Enter key to run tests on demand. To stop the server and quit dev mode, use Ctrl-C or type 'q' and press the Enter key.
[INFO] Source compilation was successful.
[INFO] Tests compilation was successful.
[INFO] [AUDIT   ] CWWKT0017I: Web application removed (default_host): http://192.168.100.167:9085/
[INFO] [AUDIT   ] CWWKZ0009I: The application inventory has stopped successfully.
[INFO] **[err]** SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[INFO] [**err**] SLF4J: Defaulting to no-operation (NOP) logger implementation
[INFO] **[err]** SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[INFO] [AUDIT   ] CWWKT0016I: Web application available (default_host): http://192.168.100.167:9085/
[INFO] [AUDIT   ] CWWKZ0003I: The application inventory updated in 0.421 seconds.
```

**Comment by jj-zhang** [comment]
LGTM
